Editor/BacktraceXcodePostBuild.cs

```diff
@@ -0,0 +1,117 @@
+#if UNITY_IOS
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEditor.Callbacks;
+using UnityEditor.iOS.Xcode;
+using UnityEditor.iOS.Xcode.Extensions;
+using UnityEngine;
+
+namespace Backtrace.Unity.Editor.iOS
+{
+    /// <summary>
+    /// Links Backtrace.xcframework and CrashReporter.xcframework
+    /// embeds, signs Backtrace and sets Swift/iOS settings.
+    /// </summary>
+    public static class BacktraceXcodePostBuild
+    {
+        // Runs late after post-processors
+        private const int PostBuildOrder = 500;
+
+        [PostProcessBuild(PostBuildOrder)]
+        public static void OnPostProcessBuild(BuildTarget buildTarget, string buildPath)
+        {
+            if (buildTarget != BuildTarget.iOS) {
+                 return;
+                 }
+
+            var projectPath = PBXProject.GetPBXProjectPath(buildPath);
+            var project = new PBXProject(); 
+            project.ReadFromFile(projectPath);
+
+#if UNITY_2019_3_OR_NEWER
+            string appTargetGuid = project.GetUnityMainTargetGuid();
+            string unityFrameworkTargetGuid = project.GetUnityFrameworkTargetGuid();
+#else
+            string appTargetGuid = project.TargetGuidByName("Unity-iPhone");
+            string unityFrameworkTargetGuid = appTargetGuid;
+#endif
+            if (string.IsNullOrEmpty(appTargetGuid) || string.IsNullOrEmpty(unityFrameworkTargetGuid))
+            {
+                Debug.LogError("[Backtrace] iOS post-build: could not resolve Xcode targets.");
+                return;
+            }
+
+            // Locate exported xcframeworks
+            string FindXCFramework(string folderName)
```

Comment:
can we move it to class instance? It's not popular in C# to have a function declaration inside another function. its also inconsistent with the rest of the library code